js/chat.js

```diff
@@ -581,11 +581,13 @@ function readMessage(input) {
     } else if ('message_id' === kind) {
         $('#'+obj.prompt_id).attr('id', obj.message_id); // set user prompt id
     } else if (text === '[DONE]' || text === '[LENGTH]') {
-        // End of the message
-        runStarted(false);
+        // End of the message, keep run_id for error on thread active yet
```

Comment:
If I understand the meaning correctly, the following is better phrasing. (If I've misunderstood, some other rephrasing is called for.)
```suggestion
        // End of the message, keep run_id for error on thread remaining active
```